Simplenote/Classes/SPOptionsViewController.m

```diff
@@ -313,6 +350,15 @@ - (UITableViewCell *)tableView:(UITableView *)tableView cellForRowAtIndexPath:(N
                     cell.accessoryView = self.biometrySwitch;
                     cell.tag = kTagTouchID;
```

Comment:
Missing break on this one `break;`.

(ObjC's behavior is to fall thru unless there's a break)